tools/build_container.sh

```diff
@@ -0,0 +1,23 @@
+curl -sL https://a01tools.blob.core.windows.net/droid/linux/a01droid -o tools/a01droid
+chmod +x tools/a01droid
```

Comment:
You should be able to add the `a01droid` to the docker directly in the docker file using `ADD` command.

Reference: https://docs.docker.com/engine/reference/builder/#add